wagtail/search/backends/elasticsearch8.py

```diff
@@ -35,5 +37,36 @@ class Elasticsearch8SearchBackend(Elasticsearch7SearchBackend):
     autocomplete_query_compiler_class = Elasticsearch8AutocompleteQueryCompiler
     results_class = Elasticsearch8SearchResults
 
+    def _get_host_config_from_url(self, url):
+        """Given a parsed URL, return the host configuration to be added to self.hosts"""
+        use_ssl = url.scheme == "https"
+        port = url.port or (443 if use_ssl else 80)
+
+        # the verify_certs and http_auth options are no longer valid in Elasticsearch 8
```

Comment:
Given that Elasticsearch 8 is more picky than earlier versions about what is and isn't included in this dict, it seemed slightly neater to have a single point in the code that defines what we're passing to Elasticsearch, rather than having that information spread across multiple files.